### ISSUE TITLE
indent: queue.in

### ISSUE THREAD

**Issue by jbodah** [elixir-lang/vim-elixir#300]
# Actual

```ex
def handle_call({:get_in_line_for_lock, key}, from, state) do
  queue = state[:queues][key] || :queue.new
  queue = queue.in(from, queue)
    hello
end
```

# Expected

```ex
def handle_call({:get_in_line_for_lock, key}, from, state) do
  queue = state[:queues][key] || :queue.new
  queue = queue.in(from, queue)
  hello
end
```

**Comment by jbodah** [comment]
Waiting on fix to be pushed

**Comment by jbodah** [comment]
Fixed in 222381d
